opg-php-fpm-71-ppa-1604/docker/my_init.d/40-opg_php_xdebug_remote_host

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+rm -f /etc/php/7.1/fpm/conf.d/*xdebug*
```

Comment:
is this the correct path for the php fpm install? normally it lives under php/major_version_number/fpm

Reply:
yea, i can confirm, expected files are under the same path in containers build from this image 
```
root@d21ef9abbc89:/app# ls -la /etc/php/7.1/
total 28
drwxr-xr-x 11 root root 4096 Nov 17 13:41 .
drwxr-xr-x 10 root root 4096 Nov 17 13:41 ..
drwxr-xr-x  5 root root 4096 Nov 17 13:41 cli
drwxr-xr-x  7 root root 4096 Nov 17 13:41 fpm
drwxr-xr-x  2 root root 4096 Nov 17 13:30 mods-available
-rw-r--r--  1 root root   98 Nov 16 14:10 xdebug-disable.ini
-rw-r--r--  1 root root  334 Nov 16 14:10 xdebug-enable.ini
```